addon/templates/components/nypr-story/byline.hbs

```diff
@@ -0,0 +1,21 @@
+<footer class="text text--medium text--dotted" id="byLine">
+  <time datetime="{{moment-format dateLineDatetime 'YYYY-MM-DD'}}">
+    {{moment-format dateLineDatetime 'MMM D, YYYY'}}
```

Comment:
the passed in attribute is actually the `newsdate`. I updated the story template to use that attribute but kept the scoped name in the component as is. Would you update `dateLineDatetime` to be `newsdate` and also update the app templates to use that name as well?